quinn/src/tests.rs

```diff
@@ -120,7 +128,7 @@ fn read_after_close() {
             .unwrap()
             .await
             .expect("connect");
-        tokio::time::delay_until(Instant::now() + Duration::from_millis(100)).await;
+        tokio::time::sleep_until(Instant::now() + Duration::from_millis(100)).await;
```

Comment:
This is equivalent to
```suggestion
        tokio::time::sleep(Duration::from_millis(100)).await;
```

Reply:
Doesn't look like anything was done here. Was the decision to leave it for future work as an unrelated refactoring?